identity/What-We-Own.md

```diff
@@ -0,0 +1,12 @@
+# What repositories we manage
+
+## Current owned services in alphabetical order
+| Name | Description |
+| ---- | ----------- |
+| [Bard](https://github.com/DataBiosphere/bard) | Bard is a service which manages our integrations with metrics services. As of 6/24/2020, this is only Mixpanel. |
+| [Bueller](https://github.com/DataBiosphere/terra-ui/blob/dev/integration-tests/Bueller.md) | Bueller is a service which hosts our integration tests for the Terra UI as an application, with one endpoint for each integration test. |
+| [Calhoun](https://github.com/DataBiosphere/calhoun) | Calhoun is a service which generates read-only views of notebooks for previews. It does not require a running cluster, and spits out the notebook file as markdown. |
+| [Job Manager](https://github.com/DataBiosphere/job-manager) | Job manager is a UI and API service that is ued by users of the Terra platform to manage and inspect in progress, finished, or errored batch jobs. |
+| [Lyle](https://github.com/DataBiosphere/lyle) | Lyle is a service to manage allocating temporary google service accounts for our test users |
```

Comment:
Nit: "for our test users" sounds a little like it creates service accounts for existing users (like pets). Maybe something like this?
```suggestion
| [Lyle](https://github.com/DataBiosphere/lyle) | Lyle is a service to manage allocating temporary google service accounts for use as test users |
```